bot/commands/Configuration/aliasInhibitor.js

```diff
@@ -0,0 +1,18 @@
+const { Inhibitor } = require('klasa');
+
+module.exports = class extends Inhibitor {
+
+    constructor(...args) {
+        super(...args, {
+            name: 'aliasInhibitor'
+        });
+    }
+
+    async run(msg, cmd) {
+        const [alias] = message.content.slice(msg.prefixLength).trim().split(' ');
```

Comment:
I stole this from Klasa discord and I'm not familiar enough with JS to know if this would correctly get the alias used.